src/games/strategy/triplea/ai/proAI/util/ProUtils.java

```diff
@@ -46,6 +47,26 @@ public ProUtils(final ProAI ai) {
     return unitTerritoryMap;
   }
 
+  public List<PlayerID> getEnemyPlayersInTurnOrder(final PlayerID player) {
```

Comment:
True. Though there are other problems as well. Having all the utilities as objects is causing a giant web of parameters to pass around. I think what I want to do is create a sort of static class to hold the game data and static parameters around it such as settings, players, etc (almost how the AI would translate the game.xml and current game data).